core/src/main/scala/com/salesforce/op/stages/impl/preparators/SanityCheckerMetadata.scala

```diff
@@ -179,6 +179,9 @@ case class SummaryStatistics
     meta.build()
   }
 
+  private[op] def +(sum: SummaryStatistics): SummaryStatistics = new SummaryStatistics(count, sampleFraction,
```

Comment:
you should be able to use `com.twitter.algebird.macros.caseclass` instead. i.e.
```scala
import com.twitter.algebird.Operators._
import com.twitter.algebird.macros.caseclass
implicit val summaryStatisticsSG = caseclass[SummaryStatistics]

val (s1: SummaryStatistics, s2: SummaryStatistics) = ???
s1 + s2
```
and same with `Correlations` type as well.

Reply:
but I dont want the count or sample fraction to be added...